apps/prairielearn/src/components/PageFooter.tsx

```diff
@@ -0,0 +1,62 @@
+                <path class="cls-1" d="M30.62,5.14c.1-.09,2.56-2.12,6.56,.28,.08,.05,.17,.07,.26,.07,.17,0,.34-.09,.43-.24,.14-.24,.06-.54-.17-.69-4.64-2.78-7.6-.28-7.73-.17-.21,.18-.23,.49-.05,.7,.18,.21,.5,.23,.7,.05Z" />
+              </svg>
+            </a>
+          </div>
+          <div class="col-6 d-flex justify-content-end">
+            <a
+              href="https://prairielearn.com"
+              class="text-decoration-none me-3 page-footer-text d-none d-md-inline"
+            >
+              Company
+            </a>
```

Comment:
What's the rationale for having both the logo and "Company" text linking to the same place?